KubeArmor/core/kubeUpdate.go

```diff
@@ -48,19 +48,35 @@ func (dm *KubeArmorDaemon) HandleNodeAnnotations(node *tp.Node) {
 		node.Annotations["kubearmor-policy"] = "enabled"
 	}
 
-	if lsm, err := os.ReadFile("/sys/kernel/security/lsm"); err == nil {
-		if !strings.Contains(string(lsm), "apparmor") && !strings.Contains(string(lsm), "selinux") {
-			// exception: neither AppArmor nor SELinux
-			if node.Annotations["kubearmor-policy"] == "enabled" {
-				node.Annotations["kubearmor-policy"] = "audited"
-			}
+	// == LSM == //
+	var lsm []byte
+	var err error
+
+	// Check if enforcer is set in the node annotations
+	if v, ok := node.Annotations["kubearmor.io/enforcer"]; ok {
+		lsm = []byte(v)
+	} else { // Read the lsm from the system
+		lsm, err = os.ReadFile("/sys/kernel/security/lsm")
+		if err != nil {
+			kg.Errf("Failed to read /sys/kernel/security/lsm (%s)", err)
+		}
```

Comment:
Everything looks good.
Small NIT: Instead of doing string conversion again and again we can make `lsm` object string and convert the `byteData` to string after reading from file. Also, some error handling if lsm file is empty:

```suggestion
		lsmByteData, err = os.ReadFile("/sys/kernel/security/lsm")
		if err != nil {
			kg.Errf("Failed to read /sys/kernel/security/lsm (%s)", err)
		} else if len(lsmByteData) == 0 {
     kg.Errf("Failed to read /sys/kernel/security/lsm: empty file")
   }

   lsm = string(lsmByteData)
```